src/visible.js

```diff
@@ -102,10 +102,6 @@ eg.module("visible", ["jQuery", eg, document], function($, ns, doc) {
 				delay = -1;
 			}
 
-			if (typeof delay === "undefined") {
-				delay = -1;
-			}
-
```

Comment:
@cimfalab Could you change isInteger code?
Number.isInteger supports low coverage.
https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger